src/main/java/org/symphonyoss/symphony/messageml/elements/TableBody.java

```diff
@@ -43,4 +44,26 @@ public void validate() throws InvalidInputException {
     assertNoText();
     assertContentModel(Collections.<Class<? extends Element>>singleton(TableRow.class));
   }
-}
+
+
+  @Override
+  public void asPresentationML(XmlPrintStream out) {
+    Element parent = getParent();
+
+    if (!TableSelect.class.equals(parent.getClass())) {
```

Comment:
Instead of doing this Parent thing, why don't we actively implement new classes for the tbody within a formSelect?